### PR TITLE
Use Cassandra Admin in MultiStorage integration test

### PR DESCRIPTION
**Pull request by Torch3333**
This refactor the MultiStorage integration test classes to rely on the Cassandra Admin instead of managing schema using cqlsh. Thanks to that, cqlsh is not needed anymore by the integration testing so I removed it from the CI configuration.

### CODE REVIEW

File: core/src/integration-test/java/com/scalar/db/storage/multistorage/MultiStorageAdminIntegrationTest.java [hunk 1]
```diff
@@ -48,6 +47,98 @@ public static void setUpBeforeClass() throws Exception {
...
+    cassandraAdmin.createNamespace(NAMESPACE2);
+    TableMetadata tableMetadata =
+        TableMetadata.newBuilder()
+            .addPartitionKey("c1")
+            .addClusteringKey("c4")
+            .addColumn("c1", DataType.INT)
+            .addColumn("c2", DataType.TEXT)
+            .addColumn("c3", DataType.INT)
+            .addColumn("c4", DataType.INT)
+            .addColumn("c5", DataType.BOOLEAN)
```
Comment (brfrn169) [inline core/src/integration-test/java/com/scalar/db/storage/multistorage/MultiStorageAdminIntegrationTest.java:67]:
```suggestion
        TableMetadata.newBuilder()
            .addPartitionKey(COL_NAME1)
            .addClusteringKey(COL_NAME4)
            .addColumn(COL_NAME1, DataType.INT)
            .addColumn(COL_NAME2, DataType.TEXT)
            .addColumn(COL_NAME3, DataType.INT)
            .addColumn(COL_NAME4, DataType.INT)
            .addColumn(COL_NAME5, DataType.BOOLEAN)
```

**Review by brfrn169 — COMMENTED** [review]
LGTM! Left a very minor suggestion. Please have a look at it. Thank you!

**Review by feeblefakie — DISMISSED** [review]
LGTM! Thank you!

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**